### PR TITLE
CMCL-1349: Impulse sample scene

### PR DESCRIPTION
**Pull request by gaborkb**
### Purpose of this PR
Upgraded our old ImpulseWave sample scene. Added new art, cleaned up scene, cleaned up and renamed old code, materials set.
Replaced Canvas UI with SamplesToggleGUI and SamplesButtonGUI scripts, so no dependency on input modules that were not generic between UI Toolkit and built-in.

### Testing status
- [ ] Added an automated test
- [ ] Passed all automated tests
- [x] Manually tested 

### Documentation status

- [ ] Updated [CHANGELOG](https://keepachangelog.com/en/1.0.0/)
- [ ] Updated README (if applicable)
- [ ] Commented all public classes, properties, and methods
- [ ] Updated user documentation

### Package version

### CODE REVIEW

File: com.unity.cinemachine/Samples~/3D Samples/Impulse/Impulse Wave.unity [hunk 1]
```diff
@@ -0,0 +1,758 @@
+%YAML 1.1
```
Comment (glabute) [inline com.unity.cinemachine/Samples~/3D Samples/Impulse/Impulse Wave.unity:1]: This should be in the game view on a canvas

image

File: com.unity.cinemachine/Samples~/3D Samples/Impulse/Impulse Wave.unity [hunk 2]
```diff
@@ -0,0 +1,758 @@
...
+%TAG !u! tag:unity3d.com,2011:
+--- !u!29 &1
```
Comment (glabute) [inline com.unity.cinemachine/Samples~/3D Samples/Impulse/Impulse Wave.unity:3]: Help text should be fixed to describe the canvas way of invoking the impulse, and impulse listener on the vcam should be mentioned

**Review by glabute — CHANGES_REQUESTED** [review]
I pushed a commit reducing the number of cubes.  Performance was embarrassing.  Unity doesn't behave well when 2000 update methods are called every frame.

File: com.unity.cinemachine/Samples~/3D Samples/Impulse/Impulse Wave.unity [hunk 3]
```diff
@@ -38,7 +38,7 @@ RenderSettings:
   m_ReflectionIntensity: 1
```
Comment (glabute) [inline com.unity.cinemachine/Samples~/3D Samples/Impulse/Impulse Wave.unity:38]: This object:
image

image

Which gives this when old input system is disabled:

image

Is there a way to do this in a way that works out of the box with both old and new input systems?

**Review by glabute — CHANGES_REQUESTED** [review]

File: com.unity.cinemachine/Samples~/3D Samples/Impulse/Impulse Wave.unity (hunk 3 above)
Comment (gaborkb) [reply com.unity.cinemachine/Samples~/3D Samples/Impulse/Impulse Wave.unity:38]: Fixed it in PR (https://github.com/Unity-Technologies/com.unity.cinemachine/pull/758) that targets this PR.

**Review by glabute — APPROVED** [review]

**Pull request merged**